first-step-spring-class-config/src/main/java/org/elearning/portfolio/First.java

```diff
@@ -0,0 +1,59 @@
+package org.elearning.portfolio;
+
+
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+import org.elearning.portfolio.user.*;
+import org.elearning.portfolio.message.*;
+import org.elearning.portfolio.services.*;
+
+import java.util.List;
+
+
+public class First {
+
+	private static ApplicationContext context;
+
+	public static void main(String[] args) {
+
+
+		context = new ClassPathXmlApplicationContext(
+				"spring-module.xml");
```

Comment:
Jeżeli konfiguracja jest w klasie zdefiniowana za pomocą adnotacji, a nie w pliku xml no to wtedy kontekst inicjalizujemy w inny sposób z wykorzystaniem klasy AnnotationConfigApplicationContext